### PR TITLE
HTMX CEOs to appear in court in Intercooler.js lawsuit

### PR DESCRIPTION
**Pull request by ChristianPavilonis**
## what does this PR do?

- [x] Adds a new article
- [ ] Fixes a bug / Implements a feature (please describe below)

## justification

- [x] If you are adding a new article, why is it funny? (please explain below)
- [ ] If you are implementing a feature, why is it important? (please explain below)

there's 1074 HTMX CEOs, both intercooler and htmx are by carson gross.

## checklist

- [x] I have read the [contributing guide](https://github.com/TehcCringe/TehcCringe/blob/master/docs/contributing.md)
- [x] I have read the [rules](https://github.com/TehcCringe/TehcCringe/blob/master/docs/rules.md)
- [x] I have read the [style guide](https://github.com/TehcCringe/TehcCringe/blob/master/docs/style-guide.md)

## would your mom approve of this PR?

- [x] Yes
- [ ] No

If you selected no, please explain why

### CODE REVIEW

**Comment by ChristianPavilonis** [comment]
Leaving as draft for now, more jokes later.

File: articles/htmx-ceos-to-appear-in-court-in-intercoolerjs-lawsuit/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - news
+  - htmx
+  - lawsuit
+author: "https://x.com/ChristianThePav"
+displayName: "Christian Pav"
+---
+
+BOSEMAN MONTANA - Carson Gross, CEO and founder of Intercooler.js is suing Carson Gross' HTMX for "stealing my ideas and claiming ownership. Unfair!"
+
+In response to the allegations Carson Gross stated, "Look at this nerd raggin'."
```
Comment (IroncladDev) [inline articles/htmx-ceos-to-appear-in-court-in-intercoolerjs-lawsuit/index.md:14]:
```suggestion
In response to the allegations Carson Gross stated, "Look at this nerd ragin'." 
```

**Review by IroncladDev — COMMENTED** [review]
typo

**Comment by IroncladDev** [comment]
whoops did not mean to force push

**Review by IroncladDev — APPROVED** [review]
Ship it

**Pull request merged**